lutris/gui/lutriswindow.py

```diff
@@ -105,10 +105,23 @@ def __init__(self, application, **kwargs):
         self.show_tray_icon = (
             settings.read_setting("show_tray_icon", default="false").lower() == "true"
         )
-
+        self.show_hidden_games = (
+            settings.read_setting("show_hidden_games", default="true").lower() == "true"
+        )
+        
         # Window initialization
         self.game_actions = GameActions(application=application, window=self)
-        self.game_list = pga.get_games(show_installed_first=self.show_installed_first)
+
+        game_list_raw = pga.get_games(show_installed_first=self.show_installed_first)
+        if self.show_hidden_games:
+            self.game_list = game_list_raw
+        else:
+            # Check if the PGA contains game IDs that the user does not
+            # want to see
+            ignores = pga.get_hidden_ids()
+            should_be_hidden = lambda game: not game["id"] in ignores
+            self.game_list = list(filter(should_be_hidden, game_list_raw))
```

Comment:
I think a list comprehension would be a better choice here, ymmv. Otherwise this PR and feature lgtm.

Reply:
Yeah, you're right. A list comprehension seems more pythonic than `list(filter(...))`

Reply:
I'm not completely sure, but it seems like CodeFactor doesn't like this approach, as it flags the commit which replaces lambdas with list comprehensions as too complex (Ignore the force push; I messed up bringing in the newest changes from master)

Reply:
We can skip using the lambda and just do "game["id"] not in ignores" directly in the comprehension. If game_list_raw was renamed to just games_raw, it shouldn't be too lengthy of a line hopefully.

Reply:
I thought that the lambda, since we can give it a name, would make the list comprehension easier to read, but could just be my own opinion. It does fit the line limit just fine.